### PR TITLE
Add support for hindi language

### PR DESCRIPTION
**Pull request by PushpenderSaini0**
Language support for Hindi , a language spoken in India.
### Here is a preview !
image

### CODE REVIEW

**Review by Baoyuantop — APPROVED** [review]

**Comment by Andanan** [comment]
You could add your name in the [translation-table](https://github.com/bndw/wifi-card#supported-languages) in `README.md`

**Comment by PushpenderSaini0** [comment]
I add my name in the `README.md` .
I saw the PR #78 here are the changes i find
1. The language in option is not as per standard because the PR is old
2. There are unwanted files `yarn.lock` and  `package-lock.json`
3. The translation is very similar to mine but i think it's just copy paste from 
     google translate hence has 2/3 words which are correct but not commonly used

Other than that everything looks good ! :+1:

File: src/i18n.js [hunk 1]
```diff
@@ -339,6 +339,33 @@ const resources = {
...
+      'wifi.name.placeholder': 'वाईफाई नेटवर्क का नाम',
+      'wifi.password': 'पासवर्ड',
+      'wifi.password.placeholder': 'पासवर्ड',
+      'wifi.password.hide': 'प्रिंट करने से पहले पासवर्ड छुपाएं',
+      'wifi.password.encryption': 'एन्क्रिप्शन',
+      'wifi.tip':
+        'अपने आप कनेक्ट होने के लिए अपने फ़ोन के कैमरे से QR कोड को स्कैन करें',
+      'wifi.alert.name': 'नेटवर्क का नाम खाली नहीं हो सकता',
+      'wifi.alert.password.length.5':
+        'पासवर्ड कम से कम 5 अक्षरों का होना चाहिए',
```
Comment (A-Tokyo) [inline src/i18n.js:361]:
```suggestion
        'पासवर्ड कम से कम ५ अक्षरों का होना चाहिए',
```

I suggest using native hindu numbers as well! ( :, you're the expert tho

File: src/i18n.js [hunk 2]
```diff
@@ -339,6 +339,33 @@ const resources = {
...
+      'wifi.alert.password.length.8':
+        'पासवर्ड कम से कम 8 अक्षरों का होना चाहिए',
```
Comment (A-Tokyo) [inline src/i18n.js:363]:
```suggestion
        'पासवर्ड कम से कम ८ अक्षरों का होना चाहिए',
```

File: src/i18n.js (hunk 1 above)
Comment (PushpenderSaini0) [reply src/i18n.js:361]: I won't suggest this change.
I have not seen this numerals being used a lot.
All the major Hindi newspaper use `1234567890` , even google when used in Hindi uses the same . 
Same with the my phone when i switch language to Hindi.

**Review by bndw — APPROVED** [review]
Awesome, thanks!

**Pull request merged**